utils/docker/publish_docker.sh

```diff
@@ -92,8 +101,10 @@ publish_docker_manifest () {
     TAGS=("${BIONIC_TAGS[@]}")
   elif [[ "$FLAVOR" == "focal" ]]; then
     TAGS=("${FOCAL_TAGS[@]}")
+  elif [[ "$FLAVOR" == "jammy" ]]; then
+    TAGS=("${JAMMY_TAGS[@]}")
   else
-    echo "ERROR: unknown flavor - $FLAVOR. Must be either 'bionic' or 'focal'"
+    echo "ERROR: unknown flavor - $FLAVOR. Must be either 'bionic', 'focal', or 'jammy'"
```

Comment:
you miss in the end the actual publishing part.

Reply:
😄 fixed, I think! Would've found out soon enough haha